Sources/TeslaSwift/Model/DriveState.swift

```diff
@@ -10,8 +10,14 @@ import Foundation
 import CoreLocation
 
 open class DriveState: Codable {
+    public enum ShiftState: String, Codable {
+        case drive = "D"
+        case parked = "P"
+        case reverse = "R"
+        case neutral = "N"
```

Comment:
I would be happy to provide a method or var on this type to also output a `title`